packages/angular/ssr/public_api.ts

```diff
@@ -14,12 +16,14 @@ export { createRequestHandler, type RequestHandlerFunction } from './src/handler
 export {
   type PrerenderFallback,
   type ServerRoute,
+  type ServerRoutesConfigOptions,
   provideServerRoutesConfig,
   RenderMode,
-  type ServerRouteAppShell,
   type ServerRouteClient,
   type ServerRoutePrerender,
   type ServerRoutePrerenderWithParams,
   type ServerRouteServer,
   type ServerRouteCommon,
 } from './src/routes/route-config';
+
+provideZoneChangeDetection();
```

Comment:
Question: How does Zone relate to this app shell refactoring? Should this maybe be in a different commit? Should Zones be more opt-in to make future Zoneless support easier?

Reply:
That’s a mistake.